app/src/main/java/com/framgia/wsm/utils/Constant.java

```diff
@@ -39,4 +39,19 @@ private Constant() {
         public static final String LEAVE_EARLY_WOMAN_M = "Leave early (woman only) (M)";
         public static final String LEAVE_EARLY_WOMAN_A = "Leave early (woman only) (A)";
     }
+
+    public static class Time {
+        public static int SEVEN_HOUR = 7;
+        public static int EIGHT_HOUR = 8;
+        public static int NIGHT_HOUR = 9;
+        public static int TEN_HOUR = 10;
+        public static int ELEVEN_HOUR = 11;
+        public static int TWELVE_HOUR = 12;
+        public static int THIRTEEN_HOUR = 13;
+        public static int FOURTEEN_HOUR = 14;
+        public static int FIFTEEN_HOUR = 15;
+        public static int SIXTEEN_HOUR = 16;
+        public static int FIFTEEN_MINUTES = 15;
+        public static int FORTY_FIVE_MINUTES = 45;
```

Comment:
WARNING: Variable 'FORTY_FIVE_MINUTES' must be private and have accessor methods.